cmd/cvc-operator/controller/cstorvolumeclaim.go

```diff
@@ -375,7 +384,10 @@ func (c *CVCController) distributeCVRs(
 	for count, pool := range usablePoolList.Items {
 		pool := pool
 		if count < pendingReplicaCount {
-			_, err = createCVR(service, volume, claim, &pool)
+			rInfo := replicaInfo{
```

Comment:
you can move it to function scoped variable

Reply:
Done